setup.py

```diff
@@ -16,7 +16,15 @@
     author_email="dev@bandprotocol.com",
     url="https://github.com/bandprotocol/bandchain",
     keywords=["BAND", "BLOCKCHAIN", "ORACLE"],
-    install_requires=["bech32", "bip32", "ecdsa", "mnemonic", "grpcio", "protobuf"],
```

Comment:
I think we can remove `setup.py` anb `requirements.txt` (I prefer using poetry too)